dspace-jspui/src/main/java/org/dspace/app/webui/components/VersioningItemHome.java

```diff
@@ -91,26 +91,24 @@ public void process(Context context, HttpServletRequest request,
 				throw new PluginException(e.getMessage());
 			}
 
-			if (latestVersion != null) {
```

Comment:
Might be warped in the github view, the removal bracket is shown in IntelliJ....